src/services/TelegramService.ts

```diff
@@ -0,0 +1,22 @@
+import nodeFetch, {RequestInit} from 'node-fetch';
+
+export class TelegramService {
+  static readonly baseUrl = `https://api.telegram.org/bot${process.env.TELEGRAM_BOT_TOKEN}`;
+                         
+  static async getMemberCount(): Promise<number> {
+    const url = `${this.baseUrl}/getChatMemberCount?chat_id=@MilliontokensOfficial`;
+    const fetchOptns: RequestInit = {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    } 
+
+    const resp = await nodeFetch(url, fetchOptns);
+    const respBody = await resp.json();
+
+    if (!respBody.ok) throw `Telegram request failed`;
+
+    return respBody.result as number;
```

Comment:
I missed it the first go around but it doesn't look like we are actually asserting that this is a number and instead "casting". In cases where we receive values from outside the current scope (or if we're crafting values within the scope) we should asserting whether or not the value is what we are expecting. Thoughts?